### PR TITLE
Bump to generated, rebuilt toolchains

### PR DESCRIPTION
**Pull request by wrowe**
Commit Message:
Ensure the checked-out envoy-build-tools repo corresponds to the regeneraged,
checked-in newly generated docker images.

Signed-off-by: William A Rowe Jr <wrowe@vmware.com>
Additional Description:
Risk Level: low
Testing: CI pipeline TBD
Docs Changes: n/a
Release Notes: n/a
Platform Specific Features: Picks up new windows paths
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by phlax — DISMISSED** [review]
wow, i really believed everything was up to date and actually nothing was 8/

we really need #19930  

lgtm, thanks @wrowe - lets hope CI agrees

**Comment by wrowe** [comment]
It may be that bazel is regenerating the toolchain on a mismatch; that would be horribly inefficient. It would be nice to derive this from the originating commit id. Not sure we can do that.

Sorry this took some time, I was trying to validate locally, but am completely entangled in a windows mismatch for my docker host and build tools image, and am unable to toggle to hyperv isolation, because that breaks bazel's ability to write symlinks on the host filesystem. Chickens and eggs.

**Comment by rojkov** [comment]
/wait
on CI

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -64,11 +64,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "envoy-build-tools",
         project_desc = "Common build tools shared by the Envoy/UDPA ecosystem",
         project_url = "https://github.com/envoyproxy/envoy-build-tools",
-        version = "1162be3669036d2c09359a95d39ff65fc6608f39",
-        sha256 = "8f5ac011a443649a27a7b82bc447de4f564f9cb5b6812d87c3bc1b1e74d2055f",
+        version = "92823669a6005a14958795356e7cf59b90472ee3",
+        sha256 = "f36baf4ff1815a0682119d09965b8aadfe67b550a9121a3d110d68e15a61fd08",
```
Comment (moderation) [inline bazel/repository_locations.bzl:68]: Can we use latest consider this isn't passing CI yet?
`version = "074db8931da210585fad66e62ee6a63095c295d4",`
`sha256 = "ab44d9c7505dcb60a24360d0cb4a6478e3766e9ed40ed3ce183557bd2dcadbcf",`
`release_date = "2022-02-16",`

**Review by moderation — CHANGES_REQUESTED** [review]
Latest commit

**Pull request closed without merging**